Extension/browser/firefox/lib/contentPolicy.js

```diff
@@ -129,12 +129,12 @@ var WebRequestHelper = exports.WebRequestHelper = {
 
```

Comment:
I think no need to remove these checks (except `if ("" + context === "[object ChromeWindow]")`). You have to create a cache for `getTabForContentWindow` function result. Something like `cache = WeakMap(winTop, tab)`